### PR TITLE
[DONT MERGE] src: dynamic block sync size.

### PR DESCRIPTION
**Pull request by 0xFFFC0000**

### CODE REVIEW

File: src/cryptonote_config.h [hunk 1]
```diff
@@ -93,11 +93,15 @@
 #define DIFFICULTY_BLOCKS_ESTIMATE_TIMESPAN             DIFFICULTY_TARGET_V1 //just alias; used by tests
 
 
-#define BLOCKS_IDS_SYNCHRONIZING_DEFAULT_COUNT          10000  //by default, blocks ids count in synchronizing
-#define BLOCKS_IDS_SYNCHRONIZING_MAX_COUNT              25000  //max blocks ids count in synchronizing
-#define BLOCKS_SYNCHRONIZING_DEFAULT_COUNT_PRE_V4       100    //by default, blocks count in blocks downloading
```
Comment (Rucknium) [inline src/cryptonote_config.h:98]: Are these whitespace changes?

File: src/cryptonote_config.h [hunk 2]
```diff
@@ -93,11 +93,15 @@
...
-#define BLOCKS_SYNCHRONIZING_DEFAULT_COUNT              5     //by default, blocks count in blocks downloading
-#define BLOCKS_SYNCHRONIZING_MAX_COUNT                  2048   //must be a power of 2, greater than 128, equal to SEEDHASH_EPOCH_BLOCKS
```
Comment (Rucknium) [inline src/cryptonote_config.h:100]: Whitespace?

File: src/cryptonote_config.h (hunk 1 above)
Comment (0xFFFC0000) [reply src/cryptonote_config.h:98]: The problem is name of `BLOCKS_SYNCHRONIZING_DEFAULT_MEDIAN_WINDOW` is too long. 

For now lets just test functionality. I will fix these when submitting to `monero-project/monero`.

File: src/cryptonote_config.h (hunk 2 above)
Comment (0xFFFC0000) [reply src/cryptonote_config.h:100]: Same as the other.

**Pull request closed without merging**